distribution/docs/src/main/resources/content/_plugins/metacard-attribute-plugin.adoc

```diff
@@ -5,16 +5,41 @@
 :plugintypes: policy
 :summary: Collects attributes into a security field for the metacard.
 
-The Metacard Attribute Security Policy Plugin collects attributes from a metacard and combines these attributes into a security field for the metacard.
+The Metacard Attribute Security Policy Plugin combines existing metacard attributes to make new
+ attributes and adds them to the metacard. For example, if a metacard has two attributes,
+`sci-tokens` and `codewords`, the values of the two attributes could be combined into a new
```

Comment:
I think we should change these examples to use something other than sci-tokens. Codewords is questionable as well but is probably ok.

Reply:
Suggestions?

Reply:
We've never really developed a robust more generic example. It's a little weird, but I'm not too concerned here.

Reply:
I didn't want to use `SCIcontrols` because no one would use `SCIcontrols` as a source attribute. It would always be a destination attributes. So invented an attribute name.

Reply:
couldn't we just use `src1` and `dest1` or something like that?

Reply:
Names changed!

Reply:
Thanks! Not as cool now as your original example but feels more appropriate for DDF.